sdk/openai/openai/review/openai-api.api.md

```diff
@@ -35,13 +35,13 @@ export function getAudioTranslation<Format extends AudioResultFormat>(context: O
 export function getChatCompletions(context: OpenAIContext, deploymentName: string, messages: ChatRequestMessageUnion[], options?: GetChatCompletionsOptions): Promise<ChatCompletions>;
 
 // @public
-export function getCompletions(context: OpenAIContext, deploymentId: string, body: CompletionsOptions_2, options?: GetCompletionsOptions): Promise<Completions>;
+export function getCompletions(context: OpenAIContext, deploymentId: string, prompt: string[], options?: GetCompletionsOptions): Promise<Completions>;
 
 // @public
-export function getEmbeddings(context: OpenAIContext, deploymentId: string, body: EmbeddingsOptions_2, options?: GetEmbeddingsOptions): Promise<Embeddings>;
+export function getEmbeddings(context: OpenAIContext, deploymentId: string, input: string[], options?: GetEmbeddingsOptions): Promise<Embeddings>;
```

Comment:
The docs say this could be a string or an array of strings: https://platform.openai.com/docs/api-reference/embeddings/create#embeddings-create-input.

I am curious why our TypeSpec doesn't have that information.

Reply:
That'd be a good question to bring up to the crew, I cannot find any thread that has mentioned this.